docs/portfolio.md

```diff
@@ -15,47 +15,48 @@ We are especially interested in projects supporting researchers, focused on rese
-  
-**[Mouser](project_mouser/about.md)** - This software is used for tracking the data of animal experiments.  
-  
-**[Pi4Micronaut](project_pi4micronaut/about.md)** - A Java library crafted for developers who aim to build IoT  applications leveraging the Raspberry Pi platform.  
-  
-**[Rerum Geolocator](project_rerum_geolocator/about.md)** - The application enables users to enhance discovery and access to digital resources through geographic visualization and annotation.  
-  
+
+**[Gallery of Glosses](./project_gallery_of_glosses/about.md)** - Interfaces to view and manage Glosses
+
+**[MeltShiny](project_meltshiny/about.md)** - automates the analysis and visualization of DNA melting curves for researchers in chemistry, biology, and genetics.
+
+**[Mouser](project_mouser/about.md)** - This software is used for tracking the data of animal experiments.
+
+**[Pi4Micronaut](project_pi4micronaut/about.md)** - A Java library crafted for developers who aim to build IoT applications leveraging the Raspberry Pi platform.
+
+**[Rerum Geolocator](project_rerum_geolocator/about.md)** - The application enables users to enhance discovery and access to digital resources through geographic visualization and annotation.
+
 **[Seeing Is Believing](project_sib/about.md)** - An educational learning tool enabling visualizing the pronunciation of Spanish words.
 
-**[Shelter Volunteers](project_shelter_volunteers/about.md)** - Simplify the process of scheduling work shifts for volunteers, and to give homeless shelters visibility into their upcoming staffing.  
-  
-**[Speech Tagging](project_saltify/about)** - This software transcribes an audio sample into a written format that is accepted by SALT software.   
-  
-**[Where's Religion Desktop](project_wheres_religion_desktop/about.md)** - Where's Religion Desktop is an enhancement on the mobile version which will have extended features.  
-  
-**[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.  
+**[Shelter Volunteers](project_shelter_volunteers/about.md)** - Simplify the process of scheduling work shifts for volunteers, and to give homeless shelters visibility into their upcoming staffing.
+
+**[Speech Tagging](project_saltify/about)** - This software transcribes an audio sample into a written format that is accepted by SALT software.
+
+**[Where's Religion Desktop](project_wheres_religion_desktop/about.md)** - Where's Religion Desktop is an enhancement on the mobile version which will have extended features.
+
+**[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.
```

Comment:
@kungfuchicken the above changes are just trivial and are added by my Prettify extension.